src/map/atcommand.cpp

```diff
@@ -3935,7 +3935,7 @@ ACMD_FUNC(reload) {
 		map_msg_reload();
 		clif_displaymessage(fd, msg_txt(sd,463)); // Message configuration has been reloaded.
 	} else if (strstr(command, "questdb") || strncmp(message, "questdb", 3) == 0) {
-		do_reload_quest();
+		quest_db.reload();
```

Comment:
Unused return value.
Either explicitly flag it as ignored or use the returned bool for error handling.

https://stackoverflow.com/questions/689677/why-cast-unused-return-values-to-void